packages/core/src/card/card.stories.mdx

```diff
@@ -0,0 +1,43 @@
+import { Meta, Props, Story, Preview, API } from '@storybook/addon-docs/blocks';
+
+<Meta title="Components/Card" />
+
+# Card
+
+## Installation
+
+```typescript
+import '@cds/core/card/register.js';
+```
+
+## Usage
+
+```html
+<cds-card>
+  <div cds-layout="vertical gap:md">
+    <div class="header" cds-layout="p-t:lg p-l:lg">
+      <p cds-text="title">Profile of John Doe</p>
+    </div>
+
+    <cds-divider></cds-divider>
+
+    <div class="body" cds-layout="p:lg">
+      <p cds-text="p1">Profile information</p>
+    </div>
+
+    <cds-divider></cds-divider>
+
+    <div class="footer" cds-layout="p:lg">
+      <cds-button> View </cds-button>
+    </div>
+  </div>
+</cds-card>
+```
+
+<Preview>
+  <Story id="card-stories--basic" />
```

Comment:
When I run storybook, this demo isn't loading for me on: http://localhost:6006/?path=/story/components-card--page

Reply:
Yea, I have to switch the `stories` and `card` places in the selector. Thank you for catching that!